server/game/cards/01-DTR/AbramGrothe.js

```diff
@@ -16,7 +15,9 @@ class AbramGrothe extends DudeCard {
                 this.game.addMessage('{0} uses {1} to lead a job to discard all Abominations and wanted dudes at {2}', context.player, this, context.target),
             onSuccess: (job, context) => {
                 const dudes = this.game.getDudesAtLocation(job.mark.gamelocation).filter(dude => dude.isWanted() || dude.hasKeyword('abomination'));
-                dudes.forEach(dude => this.game.resolveGameAction(GameActions.discardCard({ card: dude }), context));
+                if(dudes.length) {
+                    context.player.discardCards(dudes);
```

Comment:
Context should be passed to the function.
```suggestion
                    context.player.discardCards(dudes, false, () = > true, {}, context);
```
I know it does not look very nice, but it is needed in case engine has to decide if the discard can be done based on play type, or what player doing the discard.

Reply:
> I know it does not look very nice, but it is needed in case engine has to decide if the discard can be done based on play type, or what player doing the discard.

ah, ok - thanks. i was wondering about those extra arguments.

Reply:
handled :heavy_check_mark: